test/language/class/class-accessors.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class accessors
+---*/
+
+function assertAccessorDescriptor(object, name) {
+  var descr = Object.getOwnPropertyDescriptor(object, name);
+  assert.sameValue(descr.configurable, true);
```

Comment:
I disagree on assert usage actually. You could imagine horrible implementation bugs that make it return `undefined` instead of `false` or ... something ... instead of `true`.